pkg/cluster/kotsadm.go

```diff
@@ -13,6 +13,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/duration"
 	certutil "k8s.io/client-go/util/cert"
+	"k8s.io/utils/pointer"
+)
+
+const (
+	KOTSADM_RQLITE_HA_REPLICA_COUNT = 3
```

Comment:
go uses camel case constants as a convention

```suggestion
	KotsadmRqliteHAReplicaCount = 3
```